src/systems/scene_broadcaster/SceneBroadcaster.cc

```diff
@@ -1079,6 +1089,29 @@ void SceneBroadcasterPrivate::SceneGraphAddEntities(
         return true;
       });
 
+  // Projectors
+  _manager.EachNew<components::Projector, components::ParentEntity,
+    components::Pose>(
+      [&](const Entity &_entity,
+          const components::Projector *_projectorComp,
+          const components::ParentEntity *_parentComp,
+          const components::Pose *_poseComp) -> bool
+      {
+        auto projectorMsg = std::make_shared<msgs::Projector>();
+        projectorMsg->CopyFrom(
+            convert<msgs::Projector>(_projectorComp->Data()));
+        // \todo(anyone) add id field to projector msg
```

Comment:
Add issue in gz-msgs ?

Reply:
yep, ticketed issue https://github.com/gazebosim/gz-msgs/issues/341